keepLogs configuration value (#4)
suggestionsbot/suggestions-docs: docs/docs/managing-suggestions.md

Suggestions are approved via the sID. Once you have it, run the command `approve <sID|message ID>`. Afterward, series of actions will happen:

* The suggestion will update as "Approved" in the suggestions channel and its results will be logged.
* The user will receive a DM that you approved their suggestion.

> * The results of that suggestion will either be posted to the suggestion logs channel or the suggestion message will be kept with its results displayed. (dependent on [`config keepLogs`](configuration.md#keep-logs))

Reviewer: For the sake of parallel structure:

```suggestion
* The results of that suggestion will either be posted to the suggestion logs channel or be kept in the suggestion message. (dependent on [`config keepLogs`](configuration.md#keep-logs))
```

## Suggestions Logging

When a suggestion is approved or rejected, one of two things will happen, depending on if you have a suggestions logs channel set and if [`config keepLogs`](configuration.md#keep-logs) is enabled (by default, it's `false`).

- If you have no suggestion logs channel set or `config keepLogs` is true, then the suggestion message will be updated with its results and the reactions removed.

> - If you have a suggestion logs channel set and `config keepLogs` is false, The bot will delete the original suggestion message and its results posted to the set logs channel. You'll find the following information in the log message either way:

Reviewer: A small typo:
```suggestion
- If you have a suggestion logs channel set and `config keepLogs` is false, the bot will delete the original suggestion message and its results will be posted to the set logs channel. You'll find the following information in the log message either way:
```

> If you wish to set up a suggestion logs channel, you may refer to the [Getting Started](README.md#setting-up-the-bot) section of the documentation.

Reviewer: This line should appear after the information list.

Author: Should I put it after line 49?

Reviewer: No, you should put it after line 83.

Author: Got it,
